package.json

```diff
@@ -4,7 +4,7 @@
   "description": "elastic.io component that connects to Salesforce API (node.js)",
   "main": "index.js",
   "scripts": {
-    "test": "eslint & mocha spec --recursive --timeout 50000",
+    "test": "mocha spec --recursive --timeout 50000",
```

Comment:
IMPORTANT!
Please, restore `eslint` and check your code in order to comply with development standards.